Sources/Mantis/Mantis.swift

```diff
@@ -153,18 +190,56 @@ public struct CropToolbarConfig {
     var includeFixedRatioSettingButton = true
 }
 
-public class LocalizationConfig {
-    public var bundle: Bundle? = Mantis.Config.bundle
-    public var tableName = "MantisLocalizable"
+// MARK: - DialConfig
+public struct DialConfig {
```

Comment:
I don't think it is necessary to move DialConfig to `Mantis.swift` because there is a lot of code in this file already. The original DialConfig.swift is to separate code to keep `Mantis.swift` not too messy.

Also I think we should still keep the definitions of DialRotationCenterType, DialAngleShowLimitType, DialOrientation, DialTheme, and DialRotationLimitType inside DialConfig because they are part of DialConfig. Then users should use something like DialConfig.DialRotationLimitType or DialConfig.RotationLimitType (previous definition)